### PR TITLE
[ENHANCEMENT] Added progress bar to opening inventory event

### PR DESCRIPTION
**Pull request by AnishBplayz**
## Description

This is an enhancement PR.

The PR Title Describes about it.

### CODE REVIEW

**Comment by AnishBplayz** [comment]
Lint issue - As there are spaces after TAB it shows merge block, will fix it soon if necessary other wise it should be good to merge

**Review by Z3rio — CHANGES_REQUESTED** [review]
I dont believe people would actually like this to be honest.
If you really want this to get added, then it should have a config value to turn it on / off.

**Comment by AnishBplayz** [comment]
> I dont believe people would actually like this to be honest. Considering most people want the inventory to open as fast as possible.
> 
> If you really want this to get added, then it should have a config value to turn it on / off.



Noted... will add a config value to it.

**Comment by AnishBplayz** [comment]
@Z3rio I have added the Config Value to it, also the lint issue is still about inconsistent indentation `spaces followed by TAB` it should be good to merge now....

File: client/main.lua [hunk 1]
```diff
@@ -442,21 +442,47 @@ end)
...
+                    currentOtherInventory = other.name
+                end
+                SendNUIMessage({
+                	action = "open",
+                	inventory = inventory,
+                	slots = Config.MaxInventorySlots,
+                	other = other,
+                	maxweight = Config.MaxInventoryWeight,
+                	Ammo = PlayerAmmo,
+                	maxammo = Config.MaximumAmmoValues,
```
Comment (Z3rio) [inline client/main.lua:464]:
```suggestion
					action = "open",
					inventory = inventory,
					slots = Config.MaxInventorySlots,
					other = other,
					maxweight = Config.MaxInventoryWeight,
					Ammo = PlayerAmmo,
					maxammo = Config.MaximumAmmoValues,
```

Resolve linting issue

**Comment by AnishBplayz** [comment]
lint fixed :)

File: client/main.lua [hunk 2]
```diff
@@ -442,21 +442,47 @@ end)
 
 RegisterNetEvent('inventory:client:OpenInventory', function(PlayerAmmo, inventory, other)
     if not IsEntityDead(PlayerPedId()) then
-        ToggleHotbar(false)
-        SetNuiFocus(true, true)
-        if other then
-            currentOtherInventory = other.name
+        if Config.Progressbar then
+                QBCore.Functions.Progressbar('open_inventory', 'Opening Inventory...', math.random(350, 750), false, true, { -- Name | Label | Time | useWhileDead | canCancel
```
Comment (Z3rio) [inline client/main.lua:446]:
```suggestion
            QBCore.Functions.Progressbar('open_inventory', 'Opening Inventory...', math.random(350, 750), false, true, { -- Name | Label | Time | useWhileDead | canCancel
```
Fix linting

File: client/main.lua [hunk 3]
```diff
@@ -442,21 +442,47 @@ end)
...
+                    action = "open",
+                    inventory = inventory,
+                    slots = Config.MaxInventorySlots,
+                    other = other,
+                    maxweight = Config.MaxInventoryWeight,
+                    Ammo = PlayerAmmo,
+                    maxammo = Config.MaximumAmmoValues,
+                })
+                inInventory = true
+                end, function() -- Play When Cancel
```
Comment (Z3rio) [inline client/main.lua:467]:
```suggestion
            end, function() -- Play When Cancel
```
Fix linting

**Review by Z3rio — CHANGES_REQUESTED** [review]
Just minor fixes, otherwise it looks fine. I will test this ingame shortly.

**Comment by AnishBplayz** [comment]
Done @Z3rio

**Comment by AnishBplayz** [comment]
@Z3rio The branch is synced and should be good to merge now

**Comment by SKITTLE6969** [comment]
This works and has stopped all the item drop bugs!!! Thanks for this man <3

**Comment by AnishBplayz** [comment]
Good to hear that @SKITTLE6969 Buddy <3

**Comment by AnishBplayz** [comment]
@Z3rio It should be good to merge, Requesting for Review again!

**Review by AnishBplayz — COMMENTED** [review]
resolved

**Pull request closed without merging**